docker/elasticsearch/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM elasticsearch:7.16.3
+FROM elasticsearch:8.17.3
 
 ENV discovery.type=single-node
 
 ENV ES_JAVA_OPTS='-Xms512m -Xmx512m'
 
+ENV xpack.security.enabled=false
```

Comment:
[For v7.X this was disabled by default](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/security-settings.html#general-security-settings)
    
[For v8.X this is enabled by default](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html)